yarn.lock

```diff
@@ -1037,6 +1050,14 @@
     core-js-pure "^3.20.2"
     regenerator-runtime "^0.13.4"
 
+"@babel/runtime-corejs3@^7.20.1":
```

Comment:
I think if you upgrade @testing-library/react to "^14.0.0" it will avoid us having multiple versions of babel/runtime-corejs3

Reply:
This out of scope for now